sdk/powerbiembedded/arm-powerbiembedded/CHANGELOG.md

```diff
@@ -2,20 +2,20 @@
 
 ## 2.0.2 (Unreleased)
 
```

Comment:
Pls specify release date we met error https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2030896&view=logs&j=7bb69cc7-6f52-585e-8b13-5bae7c4a9fa4&t=e0579ca3-ac7c-5600-ac9e-10927bbd3b94